datastore/gcloud/aio/datastore/constants.py

```diff
@@ -61,29 +61,21 @@ class TypeName(enum.Enum):
     BLOB = 'blobValue'
     BOOLEAN = 'booleanValue'
     DOUBLE = 'doubleValue'
+    ENTITY = 'entityValue'
     INTEGER = 'integerValue'
+    KEY = 'keyValue'
     NULL = 'nullValue'
     STRING = 'stringValue'
     TIMESTAMP = 'timestampValue'
 
 
-# TODO: support more than just scalars
+# TODO: add types for geoPointValue and arrayValue
 TYPES = {
+    bool: TypeName.BOOLEAN,
     bytes: TypeName.BLOB,
     dt: TypeName.TIMESTAMP,
     float: TypeName.DOUBLE,
     int: TypeName.INTEGER,
     str: TypeName.STRING,
-    bool: TypeName.BOOLEAN,
     type(None): TypeName.NULL,
 }
-
-FORMATTERS = {
-    TypeName.TIMESTAMP: lambda d: d.strftime('%Y-%m-%dT%H:%S:%M.%f000Z'),
```

Comment:
Pretty sure the `:%S:%M` was a typo, so I swapped them when I moved this line. Same for line 88.